### ISSUE TITLE
Structs with empty fields are not generated

### ISSUE THREAD

**Issue by marcozaccari** [marco-sacchi/go2jsonc#2]
Test code:
```go
package main

import "fmt"

type Test struct {
	Test Tests
}

type Tests struct {
	TestS string // Test string
	TestI int    // Test integer
}

func TestDefaults() *Test {
	return &Test{}
}

func main() {
	fmt.Printf("%+v\n", TestDefaults())
}
```

Generation:
```sh
$ go2jsonc -type=Test
No directory specified, using current working dir.
{
	// test.Tests
	"Test": <nil>
}
```

Expected:
```sh
$ go run test.go
&{Test:{TestS: TestI:0}}
```

`Tests` struct should be generated with fields with empty values:
```
	// test.Tests
	"Test": {
		"TestS": "",
		"TestI": 0
	}
}
```

**Comment by marcozaccari** [comment]
Workaround, with explicit initialization of empty elements:
```go
func TestDefaults() *Test {
	return &Test{
		Test: Tests{},
	}
}
```

Should it be mentioned in the doc?

**Comment by marco-sacchi** [comment]
> Workaround, with explicit initialization of empty elements:
> ```go
> func TestDefaults() *Test {
> 	return &Test{
> 		Test: Tests{},
> 	}
> }
> ```
> 
> Should it be mentioned in the doc?

No, the expected results you wrote in your first post is correct.

Maybe a regression from v0.1.0.

Anyway a test also is missing for this case.

**Comment by marco-sacchi** [comment]
Fixed with #1.
